layouts/default.vue

```diff
@@ -476,6 +476,10 @@ export default defineNuxtComponent({
     },
   },
   mounted() {
+    if (process.client && window) {
```

Comment:
are you sure it's safe to have client-side-only stuff like this?

Reply:
yes its only client side code, it affects scroll behaviour